### PR TITLE
SPB-778 Bugfix: Romancal dependency uploads occuring with failed regressions

### PR DESCRIPTION
**Pull request by shahm0**
SPB-778 Added a failure threshold so that failed regressions do not trigger an update during the build process

GitHub issue, Closes #

Resolves RCAL-

**Description**

This PR addresses ...

Checklist
- [ ] Tests

- [ ] Documentation

- [ ] Change log

- [ ] Milestone

- [ ] Label(s)

### CODE REVIEW

**Review by jneu — APPROVED** [review]
lgtm; but wait for @nden to review as well before merging

**Review by nden — APPROVED** [review]
Thanks!

**Pull request merged**
